client/state/data-layer/wpcom/read/following/mine/index.js

```diff
@@ -35,9 +36,17 @@ export const subscriptionsFromApi = apiResponse => {
 		return [];
 	}
 
-	return map( apiResponse.subscriptions, subscription =>
-		omit( subscription, 'meta' )
-	);
+	return map( apiResponse.subscriptions, subscription => {
+		return omitBy( {
+			ID: Number( subscription.ID ),
+			URL: subscription.URL,
+			blog_ID: toValidId( subscription.blog_ID ),
```

Comment:
i'm realllllllly ambivalent about altering the shape of api objects, especially when it's just naming style.